pkg/components/internal/jsonpath.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 The Lokomotive Authors
```

Comment:
I think it must be top level `/internal`, like `github.com/kinvolk/lokomotive/internal` so it cannot be imported by other modules.

Reply:
But the scope of the code is not project wide, just under the component's project. Even `/pkg/components/internal` can only be imported by packages in `/pkg/components` and not by others in the `pkg`.

Reply:
Oh, TIL! This is awesome.